.github/workflows/deploy-docs.yml

```diff
@@ -17,10 +17,25 @@ jobs:
             - name : Checkout
               uses: actions/checkout@v4
 
-            - name: Install Custom Doxygen Version 
-              uses: AdarshRawat1/Install-Doxygen@v1.0
-              with:
-                version: '1.11.0'
+            - name: Install Clang and dependencies
+              run: |
+                sudo apt-get update
+                sudo apt-get install -y clang
+
+            - name: Export Doxygen version in environment
+              shell: bash
+              run: |
+                echo "DOXYGEN_VERSION=1.11.0" >> $GITHUB_ENV
```

Comment:
We can use Github action env variables for this. https://docs.github.com/en/actions/learn-github-actions/variables

Reply:
Done !!